includes/class/heygov-resource.php

```diff
@@ -49,5 +50,44 @@ public function load_apps_banner() {
 			<?php
 		}
 	}
+	function heygov_forms_shortcode( $atts = array()) {
+		$args = shortcode_atts( array(
+			'maxcolumns' => '5',
+			'department' => ''
+		), $atts );
 
+		$maxcolumns = $args['maxcolumns']; 
+		$calc_medium = $maxcolumns - 1; 
+		$department = $args['department']; 
+
+        ob_start(); 
+        // HeyGov ID
+        $heygov_id = get_option('heygov_id');
+
+		// Get any existing copy of our transient data
+		if ( false === ( $forms = get_transient( 'forms' ) ) ) {
+			// It wasn't there, so regenerate the data and save the transient
+			$forms = wp_remote_get('https://heygov-api-develop-nxb3467cgq-uc.a.run.app/'.$heygov_id.'/forms?status=public&expand=department');
+				if (is_wp_error($forms)) {
```

Comment:
if the request is an error, $forms should be `empty array`. 
Also in case of error, the following 2 lines should not be executed.
```php
if (is_wp_error...) {
  $forms = [];
} else {
  			$forms = wp_remote_retrieve_body($forms);
			$forms = json_decode($forms); 
}
```